examples/docking-protein-homotrimer/docking-protein-homotrimer.cfg

```diff
@@ -0,0 +1,127 @@
+# ====================================================================
+# Rigid-body docking example
+
+# directory in which the scoring will be done
+run_dir = "run1"
+ncores = 20
+# cns_exec = "path/to/bin/cns" # optional
+
+# molecules to be docked
+molecules =  [
+    "data/1qu9_A.pdb",
+    "data/1qu9_B.pdb",
+    "data/1qu9_C.pdb",
+    ]
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+autohis = true
+
+[rigidbody]
+ambig_fname = 'data/1qu9_whiscy_air.tbl'
+sampling = 20
+noecv = true
+ncs_on = true
+numncs = 2
+ncs_sta1_1="2"
```

Comment:
you can write `ncs_stat1_1 = "2"` if you wish, spaces are allowed, not mandatory though. But if those are numbers, they should be written without `"`. Do they need to be written to CNS with `"` despite the fact they are numbers?

Reply:
Again - if we change it we need to change as well the CNS code since currently it is decoding those strings into numbers. Actions on the Python and CNS levels must be coordinated if we change things